test/canvas.test.js

```diff
@@ -53,7 +53,7 @@ describe("Canvas", ()=>{
       expect(canvas.async).toBe(true)
 
       let promise = canvas.png
-      expect(promise).toBeInstanceOf(Promise)
+      expect(promise).not.toBeInstanceOf(Buffer)
```

Comment:
I'm not sure why, but the `Promise` class being returned by Neon is not the same class as `global.Promise`. It might be that jest is overwriting the global?